tools/scrimlet/softnpu-init.sh

```diff
@@ -37,7 +36,7 @@ z_swadm arp add "$GATEWAY_IP" "$GATEWAY_MAC"
 # Configure route to upstream gateway
 z_swadm route add 0.0.0.0/0 qsfp0/0 "$GATEWAY_IP"
 
-z_swadm port list
+z_swadm link ls
```

Comment:
Not a big problem, but just curious why we are using `ls` as a subcommand for `link`, but the other commands are still using `list`?

Reply:
Ah, nevermind, I just launched the zone to test and found:

```
root@oxz_switch:~# /opt/oxide/dendrite/bin/swadm link
swadm-link 0.1.0
Manage Ethernet links.

Links are always referenced to the switch port that contains them. This is normally done by using a "link path", which
is structured like `switch_port/link_id`, e.g., `rear0/0`.

USAGE:
    swadm link <SUBCOMMAND>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

SUBCOMMANDS:
    create      Create a new link on a switch port
    delete      Delete a link on a switch port [aliases: del]
    get         Get a single link
    get-prop    Get a property of a link
    help        Prints this message or the help of the given subcommand(s)
    list        List all links on a switch port
    list-all    List all links on all switch ports [aliases: ls] <-- tada! 🎉 
    set-prop    Set a property of a link
```

Reply:
Just an alias yeah.